doc/api/deprecations.md

```diff
@@ -95,12 +95,16 @@ methods, the `options.customFds` option is deprecated. The `options.stdio`
 option should be used instead.
 
 <a id="DEP0007"></a>
-### DEP0007: cluster worker.suicide
+### DEP0007: Replace cluster worker.suicide with worker.exitedAfterDisconnect
 
-Type: Runtime
+Type: End-of-Life
```

Comment:
We could replace all instances of `End-of-Life` with `Broken` and/or `Removed`, that might be more accurate terminology anyway (but we can also keep that for a follow-up, that might be easier).